examples/src/main/java/org/apache/spark/examples/streaming/JavaRecoverableNetworkWordCount.java

```diff
@@ -26,17 +26,14 @@
 import java.util.regex.Pattern;
 
 import scala.Tuple2;
+
 import com.google.common.io.Files;
 
 import org.apache.spark.Accumulator;
 import org.apache.spark.SparkConf;
 import org.apache.spark.api.java.JavaPairRDD;
 import org.apache.spark.api.java.JavaSparkContext;
-import org.apache.spark.api.java.function.FlatMapFunction;
-import org.apache.spark.api.java.function.Function;
-import org.apache.spark.api.java.function.Function2;
-import org.apache.spark.api.java.function.PairFunction;
-import org.apache.spark.api.java.function.VoidFunction2;
+import org.apache.spark.api.java.function.*;
```

Comment:
I was saying that these shouldn't be collapsed to a `*` import, but I don't think it's that important. We don't often use this.